modules/nodes-pods-user-namespaces-configuring.adoc

```diff
@@ -6,36 +6,17 @@
 [id="nodes-pods-user-namespaces-configuring_{context}"]
 = Configuring Linux user namespace support
 
+You can configure your pods to run in Linux user namespaces by setting the `hostUsers` parameter to `false` in the workload spec, as shown in the following procedure. 
 
-.Prerequisites
+Running workloads in user namespaces allows you to specify `RunAsAny` for Security Context Constraint (SCC) fields, such as `fsGroup`, `runAsGroup`, `runAsUser`, and `supplementalGroups`. Setting these fields to `RunAsAny` can have specific benefits described throughout the {product-title} documentation. Within a user namespace, `RunAsAny` is generally considered to be a safe practice. For extra security, you can create a new SCC specifically for workloads in user namespaces by setting the `userNamespaceLevel: RequirePodLevel` field in the SCC, which requires that the workloads run in user namespaces. To require a specific SCC, set the `openshift.io/required-scc` annotation in any workload resource that can set a pod manifest template, such as a deployment or daemon set.
```

Comment:
> Running workloads in user namespaces allows you to specify `RunAsAny` for Security Context Constraint (SCC) fields, such as `fsGroup`, `runAsGroup`, `runAsUser`, and `supplementalGroups`

this is sorta off. running in a user namespace makes it safe to run as any, but the user has to explicitly set the SCC to allow it (it's not done implicitly when RequirePodLevel is set for userNamespaceLevel)